sw/host/cryptotest/testvectors/parsers/nist_cavp_aes_gcm_parser.py

```diff
@@ -15,10 +15,29 @@
 from cryptotest_util import parse_rsp, str_to_byte_array
 
 
+def is_iv_len_invalid(iv_len) -> bool:
+    """IV length checker for AES-GCM testvectors.
+
+    According to cryptolib API implementation, the only supported
+    IV lengths are either 3 or 4 words (hex length between [24, 32]).
+    Reference: sw/device/lib/crypto/impl/aes_gcm/aes_gcm.c
+
+    """
+    return iv_len < 24 or iv_len > 32
```

Comment:
I don't think this actually matches the behavior in cryptolib; an `iv_len` between 24 and 32 would also not be accepted.
```suggestion
    return iv_len != 24 and iv_len != 32
```